docs/deployment/config_descriptions.md

```diff
@@ -413,6 +413,4 @@ apiConfigs:
 
 # Footnotes
 
-[^1]: The distribution-metrics endpoint assumes the Distribution API is deployed to a single stage.
-
 [^2]: This value is used by kes only to identify the configuration set to use and should not appear in any AWS object
```

Comment:
Are we just going to keep this a a 2? Or change it to footnote #1 since you removed 1?

Reply:
I think because of the magic of markdown it will work, but I just spaced, I'll re-number it to prevent confusion.

Reply:
8ee5284dc